docker/cli
# Don't automatically request size if `--size` was explicitly set to `false`

**- What I did**

Check if the `--size` flag was explicitly set to false (with `--size=false`) and don't automatically request the size when that's the case.

**- How I did it**

With Neovim 👀 

**- How to verify it**

Run `docker ps --format={{json .}} --size=false` and check that size isn't calculated

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**

<img width="538" alt="image" src="https://user-images.githubusercontent.com/70572044/222491595-e700a471-3c52-4799-976c-fadc37e1fdba.png">
